.github/workflows/publish.yml

```diff
@@ -51,8 +51,15 @@ jobs:
       - name: Build EVE for KVM
         run: |
           rm -rf dist
-          make LINUXKIT_PKG_TARGET=push HV=kvm eve
-      - name: Build EVE
+          make HV=kvm eve
+      - name: Build EVE for Xen
+        run: |
+          rm -rf dist
+          make eve
+      - name: Push EVE for both KVM and Xen
+        # do not push either unless both can build
```

Comment:
I like the way this is going, but this current approach will increase our build time by ~40%.

Can you please look into whether for publish action specifically we can get away with NOT removing dist in between the steps (and possibly tweaking the eve build target)?